src/events/pollVotationResults.js

```diff
@@ -6,14 +6,27 @@ module.exports = {
   async execute(message) {
     const { embeds, author, client, channelId } = message;
 
-    if (author.bot && embeds[0]?.data?.type !== 'poll_result') {
+    if (
+      author.bot &&
+      !Array.isArray(embeds[0]?.fields) &&
+      embeds[0]?.data?.type !== 'poll_result'
```

Comment:
I think that if at least one of those conditions are true we would like to return early, so maybe we should use OR, what do you think?
```suggestion
      author.bot ||
      !Array.isArray(embeds[0]?.fields) ||
      embeds[0]?.data?.type !== 'poll_result'
```